### PR TITLE
Removed MUST and SHOULDs from requirements. Fixes #168.

### LINKED ISSUE #168: Remove MUST/SHOULDs from Ecosystem explanation
The MUST/SHOULDs in the Ecosystem explanation is confusing as they are not supposed to be normative requirements on implementations, but rather expectations of the ecosystem. The text should be updated to reflect this.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/172.html" title="Last updated on May 15, 2018, 9:40 PM GMT (0b46b02)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/172/f3f8e01...0b46b02.html" title="Last updated on May 15, 2018, 9:40 PM GMT (0b46b02)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -310,52 +310,53 @@ <h3>Use Cases and Requirements</h3>
...
-machine-readable data formats for expressing <a>verifiable credentials</a> which
-can also be extended with minimal coordination.
+A <a>verifiable credential</a> is expressed in one or more standard,
+machine-readable data formats which can also be extended with minimal
+coordination.
         </li>
         <li>
-<a>Verifiable credentials</a> MUST be able to be indepenently issued, stored, and
-verified.
+<a>Verifiable credentials</a> are indepenently issued, stored,
```
Comment (dlongley) [inline index.html]: typo "indepenently" => "independently"

**Comment by David-Chadwick** [comment]
LGTM

**Comment by stonematt** [comment]
the MUST/SHOULD language was expressing details that were broadly about the ecosystem.  The test suite has more targeted requirements related specifically to the data model

LGTM

**Comment by cwebber** [comment]
LGTM

**Comment by msporny** [comment]
@burnburn said on a VCWG call:
> Remove "requirements" language

Ok, will do.

**Comment by msporny** [comment]
Made @burnburn's requested change. Merging.

**Pull request merged**